python/pyhie/allspark/model/qwen_v20_moe.py

```
@@ -466,6 +480,22 @@ def _build_graph(self, torch_cfg, derive_type):
+            for i in range(cfg.dec_layer):
+                prefix = "decoder.layer.{}.".format(i)
+                fp8_gemm_map[prefix + "attention.self.weight"] = 1
+                fp8_gemm_map[prefix + "attention.output.dense.weight"] = 1
+                if self.has_shared_expert:
+                    fp8_gemm_map[prefix + "shared_expert.gate_up_proj.weight"] = 1
+                    fp8_gemm_map[prefix + "shared_expert.down_proj.weight"] = 1
+            for op in graph.ops:
+                if (op.op_type.upper().startswith("GEMM")
+                        and op.weights
+                        and op.weights[0].name in fp8_gemm_map):
+                    quantize_gemm_fp8_blockwise(op, self.fp8_block_size)
```

Flake8 (7.3.0)

[error] 'quantize_gemm_fp8_blockwise' may be undefined, or defined from star imports: ..quantization, .model_base, .quantization_utils

(F405)